### ISSUE TITLE
Most OM users don't respond to comments / complaints about their edits

### ISSUE THREAD

**Issue by map-per** [organicmaps/organicmaps#6407]
I checket the last 25 edits in January 2023 with comments and only 5 users responded. So OM users have an expecially low response rate.

In app notifications about comments would help.

(similar to #1149)

**Comment by biodranik** [comment]
Does the OSM api allow receiving responses? If yes, then it should be easy to do. The main question would be "how often should OM check for responses", while respecting mobile internet/roaming settings...

**Comment by biodranik** [comment]
Yes, let's confirm first that email notifications for changes introduced by newly registered OM users work.

Anonymous OSM notes are not a big problem from this perspective, because they do not modify the data.

**Comment by VladimirMorozov** [comment]
Anonymous notes are a big problem for mappers since they often don't have enough information and make the notes feature of OSM less useful, as it's flooded with notes no one knows what to do about.

https://community.openstreetmap.org/t/we-dont-need-anonymous-notes/105335

**Comment by map-per** [comment]
Organic maps doesn't / shouldn't create anonymous notes (I think there are rare cases of notes being uploaded anonymous, but that's a bug (#1386))
